src/pages/Register.jsx

```diff
@@ -0,0 +1,75 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+import BasicInfo from '../components/BasicInfo';
+import Skills from '../components/Skills';
+
+const StyledFieldset = styled.fieldset`
+  display: flex;
+  flex-direction: column;
+  legend {
+    font-size: 3rem;
+    font-weight: 600;
+  }
+  p {
+    font-size: 1rem;
+  }
+  .red {
+    color: red;
```

Comment:
This color just fails the contrast ratio check.
Can you use a different color?

<img width="260" alt="Screen Shot 2021-07-19 at 10 20 35 AM" src="https://user-images.githubusercontent.com/9326234/126184769-c6d4f1a9-c72d-4c8e-a012-a4f4c2f14b2b.png">